secret-rotator/svckey/svckey.go

```diff
@@ -92,12 +101,16 @@ func (p *Provisioner) Deactivate(labels map[string]string, version string) error
 	// the reason for the prefix "v" is that Secret Manager labels need to begin with a lowwer case letter
 	name := fmt.Sprintf("projects/%s/serviceAccounts/%s@%s.iam.gserviceaccount.com/keys/%s", labels["project"], labels["service-account"], labels["project"], labels["v"+version])
 
-	_, err := p.Service.Projects.ServiceAccounts.Keys.Delete(name).Do()
-	if err != nil {
-		return fmt.Errorf("Projects.ServiceAccounts.Keys.Delete: %v", err)
-	}
+	if p.enableDeletion {
+		_, err := p.Service.Projects.ServiceAccounts.Keys.Delete(name).Do()
+		if err != nil {
+			return fmt.Errorf("Projects.ServiceAccounts.Keys.Delete: %v", err)
+		}
 
-	klog.V(2).Infof("Deactivated ver. %s: %s ", version, name)
+		klog.V(2).Infof("Deactivated ver. %s: %s ", version, name)
+	} else {
+		klog.V(2).Infof("Should deactivate ver. %s: %s ", version, name)
```

Comment:
Maybe roughly "did not deactivate ver. because"? Should is not quite as telling

Reply:
That's true. Maybe change to `klog.V(2).Infof("Did not deactivate ver. %s: %s, because flag --enable-deletion is not set", version, name)` to point to the `--enable-deletion` flag?